manifest.json

```diff
@@ -440,6 +440,9 @@
             {
               "path": "./guides/deployments/podman.md"
             },
+            {
+              "path": "./guides/deployments/proxy.md"
```

Comment:
I'm open to suggestions for where to put this guide or what to name it - please feel free to rename it and push to this branch.

Also, it seems that this list is not alphabetically ordered (`postgres` would come after `podman` in that case), but I assume that's intentional?

Reply:
It was an accident :) I updated the list to be alphabetical

Reply:
Cool, thanks for confirming!